templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -1,6 +1,6 @@
-require_relative '../node_modules/react-native/scripts/react_native_pods'
-require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
-require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+require File.join(`node --print "require.resolve('react-native/package.json')"`, "../scripts/react_native_pods")
```

Comment:
i wonder if cocoapods would let us upstream some function that wraps this logic 🤔

Reply:
I guess that's not going to be easy as it bases directly on some specific NodeJS mechanism, namely dependency resolving algorithm 🤔

Reply:
yeah it might be hard, but people working on cocoapods are sympathetic to react-native :)